db/app.js

```diff
@@ -1,16 +1,17 @@
 const express = require("express");
 const app = express();
-const getTopics = require("./controllers/controller");
+const { getTopics, getArticle } = require("./controllers/controller");
 const endpoints = require("../endpoints.json");
 
-app.get("/api", (req, res, next) => {
+app.get("/api/", (req, res, next) => {
   res.status(200).send({ endpoints });
 });
 
+app.get("/api/articles/:article_id", getArticle);
 app.get("/api/topics/", getTopics);
 
-app.all("/api/*", (req, res) => {
-  res.status(404).send({ msg: "No topics found" });
+app.use((req, res) => {
```

Comment:
This setup here doesn't seem to handle 404, as it's coming from Express, but you could have the app.all() with the right message here or give this app.use() a path with '*'.